docs/pages/config-plugins/introduction.mdx

```diff
@@ -1,59 +1,62 @@
+## Why use a config plugin
 
-You can think of plugins like a bundler for native projects, and running `npx expo prebuild` as a way to bundle the projects by evaluating all the project plugins. Doing so will generate **android** and **ios** directories. These directories can be modified manually after being generated, but then they can no longer be safely regenerated without potentially overwriting manual modifications.
+Config plugins can add native configuration to your project that isn't included by default. They can be used to generate app icons, set the app name, configure **AndroidManifest.xml** and **Info.plist**, and so on.
 
-## Use a config plugin
+Generally, in CNG projects, it is best to avoid modifying these native projects manually because you cannot regenerate them safely without potentially overwriting manual modifications. Config plugins allow you to modify these native projects in a _predictable way_ by keeping track of the changes you made to the native projects and only applying them when you run `npx expo prebuild` (either manually or automatically during a CI/CD process). For example, when you change the name of your app in app config and run `npx expo prebuild`, the name will change in your native projects automatically without the need to manually update **AndroidManifest.xml** and **Info.plist** files.
 
-Expo config plugins mostly come from Node.js modules. You can install them just like other packages in your project.
+## Mods
 
-For example, `expo-camera` has a plugin that adds camera permissions to the **AndroidManifest.xml** and **Info.plist**. To install it in your project, run the following command:
+Config plugins use **mods** (short for modifiers) to modify native project files during the prebuild process. Mods are asynchronous functions that allow you to make changes to platform-specific files such as **AndroidManifest.xml** and **Info.plist**, and other native configuration files without having to manually edit them. They accept a config and a data object, then modify and return both them as a single object. For example, in native projects, `mods.android.manifest` modifies **AndroidManifest.xml** and `mods.ios.plist` modifies **Info.plist**
 
-<Terminal cmd={['$ npx expo install expo-camera']} />
+**You don't have to use mods as top-level functions (for example `with.android.manifest`) directly in your config plugin.** When you need to use a mod, you use _mod plugins_ in your config plugins. These mod plugins are provided by the `expo/config-plugins` library and wrap top-level mod functions and behind the scenes they perform various tasks. To see a list of available mods, check out the [mod plugins provided by `expo/config-plugins`](/config-plugins/mods/#mod-plugins).
 
-In your [app's config](/versions/latest/config/app/), you can add `expo-camera` to the list of plugins:
+## Characteristics of a plugin
```

Comment:
When we say "plugin functions" here, are we talking about the plugin itself, or the functions inside the plugin? Even though a plugin function could be its own plugin, I think it's good to distinquish these a bit because most folks refer to using "plugins" and it's not even obvious that it is a function in most casual usage.